src/main/java/nl/tudelft/ti2206/group9/renderer/GroupLightRenderer.java

```diff
@@ -8,6 +8,7 @@
 /**
  * @author Robin
  */
```

Comment:
Add a Javadoc description of the class. 

I know the GroupLightRenderer was already here, but please add it for symmetry of all renderers.

<documentation,review